Updates some sprites, adds missing and new handheld overlays, some fixing, and other stuff.

MrAsbestos opened the pull request:
### Purpose
This PR adds new handheld overlays, fixes various issues, and adds some belt overlays, check it out in the changelog.

### Changelog:
CL: Added a script that makes an error when it detects an empty sprite SO (orginally made by @ThatDan123, thanks Dan!)
CL: Added missing and new handheld first aid kit overlays
CL: Fixed some issues with handheld overlays for drinks
CL: Added some handheld overlays to some snack items
CL: Fixed soda cans being able to be breakable (lol)
CL: Added handheld overlays to basic treatment items
CL: Fixed EVA helmets showing up as syndicate helmets in their handheld overlays
CL: Added missing handheld overlays for donuts
CL: Added the correct handheld overlay for broken bottles
CL: New SM sprite
CL: New Stun Baton sprite
CL: Added missing emergency oxygen tank overlay
CL: Added a sprite for the double-emergency oxygen tank
CL: Added belt overlays for the stun baton, classic baton, flashlight, health analyzer, atmospheric analyzer, and flare
CL: New jukebox sprite, and also added a name, description, price and a centcom message
CL: Added box handheld overlays
CL: Added handheld overlays for normal wrapping paper
CL: Extended the shake distance for all explosions, expect more intense explosions ;)

And that's all folks!

MrAsbestos committed:
Fixes some bottle sprites issues, adds first aid kit handheld overlays and other fixes

-Added missing and new handheld first aid kit overlays
-Fixed some issues with handheld overlays for drinks
-Added some handheld overlays to some snack items
-Fixed soda cans being able to be breakable (lol)
-Added handheld overlays to basic treatment items
-Fixed EVA helmets showing up as syndicate helmets in their handheld overlays
-Added missing handheld overlays for donuts
-Added the correct overlay for broken bottles

MrAsbestos committed:
Update some sprites, added some belt overlays, and other stuff

-New SM sprite
-New Stun Baton sprite
-Added missing emergency oxygen tank overlay
-Added a sprite for the double-emergency oxygen tank
-Added belt overlays for the stun baton, classic baton, flashlight, health analyzer, atmospheric analyzer, and flare
-Added a script the makes an error when it detects an empty sprite SO (orginally made by ThatDan, thanks Dan!)
-New jukebox sprite, and also added a name, description, price and a centcom message
-Made the export scanner look less like a gun
-Added box handheld overlays
-Added handheld overlays for normal wrapping paper

MrAsbestos committed:
Extended explosions shake distance

corp-0 commented on UnityProject/Assets/Scripts/Core/Editor/EditorDrawPreview.cs:
```diff
@@ -47,6 +47,11 @@ private static void DrawProjectItem(string guid, Rect selectionRect)
+
+				if (mainTex == null)
+				{
+					Debug.LogError($"Sprite SO {sTing} has null value");
```
Use this instead
```suggestion
					Logger.LogError($"Sprite SO {sTing} has null value");
```

MrAsbestos replied:
Done, going to do some other changes, and I'll commit

MrAsbestos committed:
Some little touches

-Made the export scanner look like a gun again (for shitz and giggles trololo)
-Some changes to the missing sprite SO script

MrAsbestos committed:
Some changes in the sprite handler

this should fix it
